save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/TestSuitesController.kt

```diff
@@ -28,8 +29,10 @@ typealias ResponseListTestSuites = ResponseEntity<List<TestSuiteDto>>
 @RestController
 class TestSuitesController(
     private val testSuitesService: TestSuitesService,
-    private val scheduler: Scheduler,
 ) {
+    @Autowired(required = false)
+    private lateinit var quartzScheduler: Optional<Scheduler>
```

Comment:
why it's optional now?

Reply:
I wanted to create this bean only in certain profiles, but reverted these changes eventually. I've changed it to be required here too.